### PR TITLE
Add a Case Insensitive Option to Manifest Diff

### PR DESCRIPTION
**Pull request by bencap**
Adds an option `--ignore-case` to the manifest diff ID3C command. Running the command without this option retains the current behavior, which is to do a deep diff of manifest files while remaining sensitive to string case. Running the command with this option will perform the same diff, but will treat upper and lowercase strings the same.

### CODE REVIEW

**Comment by bencap** [comment]
Should be merged alongside both https://github.com/seattleflu/backoffice/pull/362 and https://github.com/seattleflu/id3c-customizations/pull/357

File: lib/id3c/cli/command/manifest.py [hunk 1]
```diff
@@ -403,9 +403,13 @@ def _parse(*,
     metavar = "<manifest-b.ndjson>",
     type = LocalOrRemoteFile("r"))
 
+@click.option("--ignore-case",
+    metavar = "<ignore-case>",
+    is_flag = True)
```
Comment (davereinhart) [inline lib/id3c/cli/command/manifest.py:408]: Could also add `default=False` here.

File: lib/id3c/cli/command/manifest.py [hunk 2]
```diff
@@ -403,9 +403,13 @@ def _parse(*,
...
+
 @format_doc(PROVENANCE_KEY = PROVENANCE_KEY)
 
-def diff(manifest_a, manifest_b):
+def diff(manifest_a, manifest_b, ignore_case):
```
Comment (davereinhart) [inline lib/id3c/cli/command/manifest.py:412]: A default `ignore_case=False` could also be added here, but may not be necessary if this function is only being called via click command.

**Review by davereinhart — APPROVED** [review]

File: lib/id3c/cli/command/manifest.py (hunk 2 above)
Comment (bencap) [reply lib/id3c/cli/command/manifest.py:412]: Ah true, let me add it in here just in case.

**Review by davereinhart — APPROVED** [review]
Looks good!

**Comment by bencap** [comment]
Deployed

**Pull request merged**